package.json

```diff
@@ -100,22 +100,23 @@
     "compression": "^1.7.0",
     "connect-history-api-fallback": "^1.3.0",
     "express": "^4.15.3",
-    "long": "^3.2.0",
+    "long": "^4.0.0",
     "minimist": "^1.2.0",
-    "mobx": "^3.3.1",
-    "mobx-react": "^4.3.3",
+    "mobx": "^4.1.0",
+    "mobx-react": "^5.0.0",
+    "mobx-utils": "^4.0.0",
     "nuclearnet.js": "^1.5.2",
     "protobufjs": "6.7.3",
     "react": "^16.2.0",
     "react-color": "^2.13.8",
     "react-dom": "^16.0.0",
-    "react-resize-detector": "^1.1.0",
+    "react-resize-detector": "^2.2.0",
```

Comment:
Do our hand written types for this need to be updated?

Reply:
No, version 2.0 just swapped to using a native method (ResizeObserver) rather than the hacky scroll method.